packages/rocketchat-lib/server/models/Users.js

```diff
@@ -255,6 +255,24 @@ class ModelUsers extends RocketChat.models._Base {
 		return this.update(query, update);
 	}
 
+	clearObsoleteTokens() {
+		const query = {
+			_id: { $regex: '.*' }
+		};
+
+		const update = {
+			$push: {
+				'services.resume.loginTokens': {
```

Comment:
I don't feel like this is a good enough query/update as it just assumes a lot of information about what is considered "obsolete". I think this could be handled a lot better